.eslintrc.js

```diff
@@ -22,78 +56,393 @@ module.exports = {
+    'default-case': ['warn', { commentPattern: '^no default$' }],
+    'dot-location': ['error', 'property'],
+    'dot-notation': 'error',
+    'eol-last': 'error',
+    'func-call-spacing': ['error', 'never'],
+    'func-name-matching': 'error',
+    'func-style': ['warn', 'declaration', { allowArrowFunctions: true }],
+    'generator-star-spacing': ['error', 'after'],
+    'getter-return': 'warn',
+    'guard-for-in': 'error',
+    'handle-callback-err': 'error',
+    'import/extensions': 'warn',
+    'import/first': 'error',
+    'import/no-amd': 'error',
+    'jsx-a11y/alt-text': 'warn',
+    'jsx-a11y/anchor-has-content': 'warn',
+    'jsx-a11y/anchor-is-valid': ['warn', { aspects: ['noHref', 'invalidHref'] }],
+    'jsx-a11y/aria-activedescendant-has-tabindex': 'warn',
+    'jsx-a11y/aria-proptypes': 'warn',
+    'jsx-a11y/aria-role': ['warn', { ignoreNonDOM: true }],
+    'jsx-a11y/aria-unsupported-elements': 'warn',
+    'jsx-a11y/heading-has-content': 'error',
+    'jsx-a11y/iframe-has-title': 'warn',
+    'jsx-a11y/img-redundant-alt': 'warn',
+    'jsx-a11y/no-access-key': 'warn',
+    'jsx-a11y/no-distracting-elements': 'warn',
+    'jsx-a11y/no-redundant-roles': 'warn',
+    'jsx-a11y/scope': 'warn',
+    'jsx-quotes': 'error',
+    'key-spacing': ['error', { beforeColon: false }],
+    'keyword-spacing': ['error', { before: true, after: true }],
+    'lines-between-class-members': 'error',
+    'max-depth': ['error', 5],
+    'max-nested-callbacks': ['error', 4],
+    'new-parens': 'error',
+    'no-alert': 'error',
+    'no-array-constructor': 'warn',
+    'no-await-in-loop': 'error',
+    'no-buffer-constructor': 'error',
+    'no-caller': 'error',
+    'no-catch-shadow': 'error',
+    'no-compare-neg-zero': 'error',
+    'no-cond-assign': ['warn', 'except-parens'],
+    'no-const-assign': 'warn',
+    'no-constructor-return': 'warn',
+    'no-control-regex': 'warn',
+    'no-delete-var': 'warn',
+    'no-div-regex': 'error',
+    'no-dupe-args': 'warn',
+    'no-dupe-class-members': 'warn',
+    'no-dupe-keys': 'warn',
+    'no-duplicate-case': 'warn',
+    'no-duplicate-imports': 'error',
+    'no-else-return': 'error',
+    'no-empty-character-class': 'warn',
+    'no-empty-pattern': 'error',
+    'no-eq-null': 'error',
+    'no-extra-semi': 'error',
+    'no-eval': 'error',
+    'no-ex-assign': 'warn',
+    'no-extend-native': 'error',
+    'no-extra-bind': 'error',
+    'no-extra-label': 'error',
+    // 'no-extra-parens': 'error',
+    'no-fallthrough': 'warn',
+    'no-floating-decimal': 'error',
+    'no-func-assign': 'warn',
+    'no-implied-eval': 'error',
+    'no-invalid-regexp': 'warn',
+    'no-iterator': 'error',
+    'no-label-var': 'error',
+    'no-labels': 'error',
+    'no-lone-blocks': 'error',
+    'no-loop-func': 'warn',
+    'no-mixed-operators': ['warn', {
+      groups: [['&', '|', '^', '~', '<<', '>>', '>>>'], ['==', '!=', '===', '!==', '>', '>=', '<', '<='], ['&&', '||'], ['in', 'instanceof']],
```

Comment:
This is the default value, isn't it? Is it necessary to explicitly mention it in this rule?

Reply:
I took this from the React Facebook plugin, you are right, will fix this in the next pull request.